orm/src/test/java/org/hibernate/search/test/query/facet/MultiValuedFacetingTest.java

```diff
@@ -124,6 +125,7 @@ public void stringMap() throws Exception {
 	}
 
 	@Test
+	@Ignore // HSEARCH-1927 : Range faceting on multiple numeric values does not work
```

Comment:
Yeah, it's not a good idea to commit tests not working then ignore them. It's especially painful when you try to bisect something.